### DISCUSSION TITLE
post/hexo-blog-prefetch/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#23] [Announcements]
# post/hexo-blog-prefetch/

目前本博客正在使用 Hexo 做 SSG (static site generation), 我只需要使用 Markdown 编写文章后推送到 GitHub, Cloudflare 就能自动构建出静态页面并部署到 Cloudflare Pages. 由于 Cloudflare 本身比较给力, 本博客在境外的访问速度和延迟还是不错的, PageSpeed Insights 性能得分 99. 然而由于

https://blog.lyc8503.net/post/hexo-blog-prefetch/

**Comment by miantiao-me** [comment]
https://blog.cloudflare.com/zh-cn/introducing-speed-brain/

可以看看浏览器新的 Speculation Rules API 


CF 已经在用了
